### ISSUE TITLE
Installer questions.

### ISSUE THREAD

**Issue by skittles1** [cyberjunk/meridian59-dotnet#129]
When creating the installers for Ogre client (see https://github.com/skittles1/meridian59-installers) I was figuring out how to deal with the dependencies and tried installing on a computer without the C++ redist present. Advice elsewhere says to just include the two dlls (msvcr120.dll and msvcp120.dll) in the executable folder and on my test system this was enough to satisfy the C++ redist requirement. IMO this is preferable to installing the whole redist package or redirecting the user to a website. This also prevents having to install x64 and x86 versions of C++ redist which requires more checking (and MSI doesn't allow packaging x64 stuff in the x86 installer, but the dlls themselves are fine).

Should we put these dlls back in the Engine dll folders? I've included them in the installer solution since I package them with the installer but they should probably go back into the Engine or Ogre solution too.

For .NET and DX9 the links will be provided to install these components manually (and the Ogre installer won't start without .NET 4.5.1).

One more question - in the Control Panel, did you want your name on the Ogre client install? I wasn't sure what to put here (I just put MeridianNext on the classic one as a way to identify it).

**Comment by cyberjunk** [comment]
Just a few notes:

1) Have a look at this commit, this when I removed the DLLs recently:
https://github.com/cyberjunk/meridian59-engine/commit/554dbc4abbd18cee45b9f82a01bb65a575ffad39
It seems I had at least one more DLL in the x64 folder, but I'm not exactly sure if it's still needed.

2) I think I had in mind adding a "redist" subfolder to the rootpath where "resources" and "x86"/"x64" are and then placing the necessary third party installers (DX9, .NET., VS Redistributable) there

3) I first wanted to ship the VC runtime DLLS with the package, but because there are already other installers needed (DX9, .NET), I changed my mind in favor of just installing VS runtimes like DX9/.NET...

**Comment by skittles1** [comment]
I'm not sure if we need vcomp120.dll either, I've tested on a 32-bit system without the redist and it works fine with just the two dlls (don't have a fresh 64 bit system available). I noticed a lot of other programs on my computer come with the msvcr120.dll/msvcp120.dll making me think it's a common practice to ship these two files rather than install the C++ redist (vcomp120.dll isn't in these folders).

I did want to package .NET and DX9 installers with the msi but running a bit low on time so I've just provided it as-is, still have some reading to do on the proper way to have msi run these before installing the client itself.

Both 32 and 64 bit installers are up here https://www.meridiannext.com/play/ if you want to check them out (or if you have any suggestions).
